src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -23,14 +23,16 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />
-  <ItemGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
```

Comment:
nit: more spaces at the beginning and ending of these conditions. Perhaps we should do a find/replace to clean these up across all the projects. Feel free to do that as a separate change.